compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -31,7 +31,7 @@ use std::iter;
 use std::ops::Deref;
 use std::ptr;
 
-// All Builders must have an llfn associated with them
+// All Builders must have a llfn associated with them
```

Comment:
I think it's usually pronounced ell-ell, which would mean "an" makes more sense here